umbra-js/src/types.ts

```diff
@@ -115,3 +115,9 @@ export interface UserAnnouncement {
   token: string; // token address
   txHash: string;
 }
+
+export interface SendBatch {
+  address: string;
+  token: string;
+  amount: string;
+}
```

Comment:
_(arbitrary comment location to start a thread)_

We should also add tests for all new methods (both public and helper methods). Check out `Umbra.test.ts` to see how we test the existing `send` method